pkg/sqs/sqs.go

```diff
@@ -105,21 +105,33 @@ func (s *AWSSqsSource) Read(_ context.Context, readRequest sourcesdk.ReadRequest
 		MaxNumberOfMessages: readRequestCount,
 	})
 	if err != nil {
-		log.Fatalln("Error receiving message:", err)
+		log.Printf("error receiving message:%s", err)
+		return
 	}
 	msgs := msgResult.Messages
 	for i := 0; i < len(msgs); i++ {
 		select {
 		case <-ctx.Done():
 			return
 		default:
+			s.lock.Lock()
+			var messageTimeStamp time.Time
+			atoi, err := strconv.ParseInt(msgs[i].Attributes["SentTimestamp"], 10, 64)
+			if err != nil {
+				log.Printf("error parsing sent timestamp %s", err)
+				messageTimeStamp = time.Now()
```

Comment:
In SQS, all the messages should include the well-formatted messageTimeStamp. If err, let's panic instead of putting `time.Now()`, which can mess up the watermark.

Reply:
For local tests moto is actually not giving the timestamp thats why i have added this check . Only prod sqs sends the timestamp field . Should I remove the check altogether?

Reply:
I see. Sure, let's remove the check for now.

Reply:
Assigning time.now() to the event time might mess up the watermark calculation. Instead of making the assignment, let's completely remove the error check. Please add some comments on why we're not checking and panicing, for future reference.

Reply:
ok addressed